packages/packages/v1-adapters/src/dispatchers/__tests__/index.test.ts

```diff
@@ -0,0 +1,115 @@
+import { openRoute, runCommand } from '../';
+
+describe( '@elementor/v1-adapters/dispatchers', () => {
+	beforeEach( () => {
+		( window as any ).$e = {
+			run: jest.fn(),
+			route: jest.fn(),
+		};
+	} );
+
+	afterEach( () => {
+		delete ( window as any ).$e;
+	} );
+
+	it( 'should run a V1 command that returns Promise', () => {
+		// Arrange.
+		const command = 'editor/documents/open',
+			args = { test: 'arg' };
+
+		( window as any ).$e.run.mockReturnValue( Promise.resolve( 'result' ) );
+
+		// Act.
+		const actual = runCommand( command, args );
```

Comment:
```suggestion
		const result = runCommand( command, args );
```

Reply:
Yup, that's better... 
copilot did this 😅